src/index.js

```diff
@@ -143,11 +143,13 @@ export function withField (el) {
       const { path, mapProps } = props._veeValidateConfig
       const { validations, modelValue } = toRefs(props)
       const initialValue = modelValue ? modelValue.value : undefined
+      const label = attrs.label ? attrs.label : undefined
```

Comment:
Using `attrs` will cause this to not be reactive, so when the label changes (e.g: reactive schema) it won't be reflected when a new message is generated. 

Instead you should define a `label` prop on the `withWrapperField` component:

```js
 const wrappedComponent = markRaw({
    name: 'withFieldWrapper',
    props: {
      // ....
      label: {
        type: String,
        default: undefined
      },
    },
  // ...
  }
}
```

Then you either use `toRef` or `computed` and pass that as `label` to vee-validate.


```suggestion
      const label = computed(() => props.label)
```

Reply:
Thanks for the tip @logaretm! I made the changes as suggested and added a new test case. Tested in my app as well with reactive labels (localized) and it works, reactivity is preserved 👍